std_msgs/Int16 numCtrlSteps
std_msgs/Int16MultiArray joint_mapping
std_msgs/Int16MultiArray q_cmd_radians
---
std_msgs/Bool success